Mail2Bug/Email/EWS/EWSIncomingAttachmentConverter.cs

```diff
@@ -0,0 +1,134 @@
+﻿﻿using log4net;
+using Microsoft.Exchange.WebServices.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Mail2Bug.Email.EWS
+{
+    public class EWSIncomingAttachmentConverter
+    {
+        // TFS drops large base64 inline images: http://johannblais.blogspot.com/2015/08/migrate-tfs-repro-steps-field-to.html
+        private const int MaxBase64Length = 22 * 1024; // This a safe guess
+
+        private readonly EmailMessage _message;
+
+        private string _bodyText;
+        private List<IIncomingEmailAttachment> _attachments = null;
+
+        public EWSIncomingAttachmentConverter(EmailMessage message)
+        {
+            _message = message;
+        }
+
+        public void ProcessAttachments(bool convertInlineAttachments = false)
```

Comment:
Will remove per your suggestion. You're right, there isn't a need for a default value here.